hawkbit-rest/hawkbit-mgmt-resource/src/test/java/org/eclipse/hawkbit/mgmt/rest/resource/MgmtDistributionSetResourceTest.java

```diff
@@ -1245,4 +1246,31 @@ public void searchDistributionSetMetadataRsql() throws Exception {
         return created;
     }
 
+    @Test
+    @Description("Ensures that multi target assignment through API is reflected by the repository in the case of " +
+            "DOWNLOAD_ONLY.")
+    public void assignMultipleTargetsToDistributionSetAsDownloadOnly() throws Exception {
+        final DistributionSet createdDs = testdataFactory.createDistributionSet();
+
+        // prepare targets
+        final String[] knownTargetIds = new String[] { "1", "2", "3", "4", "5" };
+        final JSONArray list = new JSONArray();
+        for (final String targetId : knownTargetIds) {
+            testdataFactory.createTarget(targetId);
+            list.put(new JSONObject().put("id", Long.valueOf(targetId)));
+        }
+        // assign already one target to DS
+        assignDistributionSet(createdDs.getId(), knownTargetIds[0], Action.ActionType.DOWNLOAD_ONLY);
+
+        mvc.perform(post(
```

Comment:
I would suggest to hardcode the url to make the test more readable and to ensure the test fails if the api breaks.
```suggestion
        mvc.perform(post("/rest/v1/distributionsets/{ds}/assignedTargets", createdDs.getId())
```